napari/layers/image/_image_key_bindings.py

```diff
@@ -38,7 +38,9 @@ def orient_plane_normal_along_view_direction(layer: Image):
     viewer = napari.viewer.current_viewer()
     if viewer.dims.ndisplay != 3:
         return
-    layer.plane.normal = layer._world_to_data_ray(viewer.camera.view_direction)
+    layer.plane.normal = layer._world_to_displayed_data_ray(
+        viewer.camera.view_direction, [-3, -2, -1]
```

Comment:
Can you use a keyword argument here? I have no idea reading this what [-3, -2, -1] is...